data-manipulation/hashing/murmur/hash-data-using-murmur3.yml

```diff
@@ -34,22 +34,20 @@ rule:
         - and:
           - description: 'hash >> 16; hash >> 13; hash >> 16'
           - count(mnemonic(shr)): 3
-          - instruction:
-            - mnemonic: shr
-            - number: 16
-          - instruction:
-            - mnemonic: shr
-            - number: 13
+          - number: 16
+          - number: 13
           - optional:
             - count(characteristic(nzxor)): 3 or more
       - and:
         # Group this two blocks under an `and` as on their own they are not
         # unique enough and would cause false positives
-        - instruction:
-          - description: k ROL r1
-          - mnemonic: rol
-          - number: 15 = r1
-        - instruction:
-          - description: hash ROL r2
-          - mnemonic: rol
-          - number: 13 = r2
+        - basic block:
```

Comment:
These were false positives before. There's no ROL 15 or ROL 13 instructions in this function